ml-proto/spec/kernel.ml

```diff
@@ -82,9 +82,9 @@ and expr' =
   | Block of expr list                      (* execute in sequence *)
   | Loop of expr                            (* loop header *)
   | Break of var * expr option              (* break to n-th surrounding label *)
-  | Br_if of var * expr option * expr       (* conditional break *)
+  | Break_if of var * expr option * expr    (* conditional break *)
+  | Break_table of var list * expr option * expr  (* indexed break *)
```

Comment:
Fair enough, but there are arguments both ways on that; not sure we should revisit the `br` naming debate (https://github.com/WebAssembly/design/issues/445) here.